tests/acceptance/application/query_test.ts

```diff
@@ -130,6 +132,267 @@ describe('Acceptance | query', function () {
           });
         });
 
+        context('when user use params', function () {
+          context('when user does not have access to param', function () {
+            it('should return a proper error with status code 422', async function () {
```

Comment:
je pense que ce devrait être une `403 - Forbidden`

Reply:
si je mets 403 j'ai le test qui rate 
`   AssertionError: expected 422 to equal 403`

Reply:
en effet pour gérer ça correctement il faut changer le code qui gère les erreurs